recipes/nmslib/all/conanfile.py

```diff
@@ -41,10 +46,19 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
+        if is_msvc(self):
+            # Not available on MSVC
+            # https://github.com/nmslib/nmslib/blob/v2.1.1/similarity_search/include/space/space_sqfd.h#L19
+            self.options.build_extras = False
```

Comment:
Instead of changing the value, it'd be better to add this step to the `validate()` function and raise an exception saying something like `"WITH_EXTRAS is not available for MSVC, please, set this option to False"`.

Reply:
That would not work too well with the default value being `True` and would cause it to be unnecessarily skipped in the CI. I simply disabled the option on MSVC instead.